README.md

```diff
@@ -299,6 +299,39 @@ const MyStyledDiv = glamorous.div(
 <MyStyledDiv /> // styles applied: {padding-top: 1, padding-right: 2, padding-bottom: 3, padding-left: 4} and anything coming from `extra-thing`.
 ```
 
+##### withProps
```

Comment:
I just had a thought. Do you think that people may get confused and try to do:

```javascript
const MyStyledInputWithProps = glamorous.input({
  padding: 10
}).withProps({type: 'text'})

// or maybe:

const MyStyledInput = glamorous.input({
  padding: 10
})

const MyStyledInputWithProps = MyStyledInput.withProps({type: 'text'})
```

In interested of that, perhaps we should support both?